scripts/update-unknown.js

```diff
@@ -8,11 +8,12 @@ async function main() {
   console.log(`${bans.length} Unknown Bans To Classify`);
   for (const ban of bans) {
     const old = ban.reason;
-    ban.reason = classifyBanReason(ban.rawReason);
+    ban.reason = classifyBanReason(ban.rawReason, ban.rawNote);
```

Comment:
As above, ban.rawNote would need to be added to the DB schema.

Reply:
Added to DB Schema in https://github.com/CommunityBanList/Communitybanlist/pull/11/commits/031fe5c50d15c57831a1b3df725a2d08fa9a098f